.changeset/rare-seas-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/backend-function': patch
+---
```

Comment:
```suggestion
---
'@aws-amplify/backend': patch
'@aws-amplify/backend-function': patch
---
```

Looks like new check is working (https://github.com/aws-amplify/amplify-backend/pull/2042)